translations/ja-JP.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
What does this translation file contribute? 

It looks like UI strings on the various pages are not externalized, so the plugin is not properly internationalized to support localization.

Reply:
> What does this translation file contribute?
@ps48 Can we remove this file?

Reply:
Yes, we can remove this.